playbooks/ansible-cloud/vcenter-appliance/files/bootstrap.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- hosts: appliance
+  gather_facts: false
+  vars:
+    ansible_password: zuul
+    ansible_user: zuul
+    authorized_keys_file:
+      esxi1: /etc/ssh/keys-zuul/authorized_keys
+      esxi2: /etc/ssh/keys-zuul/authorized_keys
+      vcenter: /home/zuul/.ssh/authorized_keys
+  tasks:
+
+    - name: lookup SSH public key
+      set_fact:
+        _ssh_key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
+
+    - name: Setup zuul user
+      authorized_key:
+        key: "{{ _ssh_key }}"
+        state: present
+        user: zuul
+        path: "{{ authorized_keys_file[inventory_hostname] }}"
+
+    - name: Setup zuul user
+      authorized_key:
+        key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCyiXfE1zHKdj6odbysr917Cn88ov0VQaPJtLKJyMNuRYAeMOFQHd50X8JO4dfZbmSo3YdJlVfz9FLRxE64mqj9bkN8hPFbkTG2F1AWXGPON5cmm4uiLPfQkWhX/LnClrhzZpNtMJYs5AEFeDs0POijcRugZsQA+wvLi0lSlhOfkqtjAJKpPUwy1wrJFDdvqdQBjpNQh/LB8c15XfQV2JT/3NX26dQe8zvHhL6NvfhBnAikodYkBr7UjSl36CBk0cPebZMZEBBiHdo76xORVkpmqDvkhFByXXeAsvRa2YWS4wxpiNJFswlRhjubGau7LrT113WMcPvgYXHYHf2IYJWD goneri.lebouder@enovance.com"
+        state: present
+        user: zuul
+        path: "{{ authorized_keys_file[inventory_hostname] }}"
```

Comment:
will need to be deleted, added when we create autoholds

Reply:
I've just removed them. How can I use the autohold thing?